modules/registry-operator-config-resources-configmap.adoc

```diff
@@ -107,4 +107,4 @@ data:
     -----END CERTIFICATE-----
 ----
 <1>  If the registry has the port, such as `registry-with-port.example.com:5000`,
-`:` should be replaced with `...`.
+`:` should be replaced with `. . .`.
```

Comment:
The dot connector should like here, `image-registry.openshift-image-registry.svc..5000`